package.json

```diff
@@ -1,29 +1,33 @@
 {
-  "name": "auro-library",
+  "name": "@aurodesignsystem/auro-library",
+  "version": "1.0.0",
```

Comment:
Do not predefine a 1.0 version release here. If anything this should be `0.0.0`. Semantic release will update this when there is an actual package version release.